runtime/calamari/src/lib.rs

```diff
@@ -371,13 +370,14 @@ impl pallet_balances::Config for Runtime {
 }
 
 parameter_types! {
-    /// Relay Chain `TransactionLengthToFeeCoeff` / 10
-    pub const TransactionLengthToFeeCoeff: Balance = mKMA / 100;
+    /// Relay Chain `TransactionLengthToFeeCoeff` / 10 (1_000_000 on Kusama)
+    pub const TransactionLengthToFeeCoeff: Balance = mKMA;
+    pub const WeightToFeeCoeff: Balance = 150_000;
```

Comment:
This is a 30x increase from before.

Reply:
could do a 10x here and rely more on the `AdjustmentVariable` to prevent attacks